MTVd-test-project/css/fonts.css

```diff
@@ -0,0 +1,41 @@
+@font-face{
```

Comment:
Font-faces count isn't equal to fonts in the fonts folder, please remove unused fonts from the folder.